### PR TITLE
Extend talent command "all"

### PR DESCRIPTION
**Pull request by natsurepo**
## Description

Please carefully read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md) before making any pull requests.
And, **Do not make a pull request to merge into stable unless it is a hotfix. Use the development branch instead.**
## Issues fixed by this PR
Added talent option to set all talents (n, e, q) at once
Modified error message to cover edge cases when player entered talent level lower than 1

## Type of changes

- [ ] Bug fix
- [ ] New feature 
- [x] Enhancement
- [ ] Documentation

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] My pull request is unique and no other pull requests have been opened for these changes
- [x] I have read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md)
- [x] I am responsible for any copyright issues with my code if it occurs in the future.

### CODE REVIEW

**Comment by TukanDev** [comment]
Wonderhoy

File: src/main/java/emu/grasscutter/command/commands/TalentCommand.java [hunk 1]
```diff
@@ -87,6 +87,30 @@ public void execute(Player sender, Player targetPlayer, List<String> args) {
...
+                    CommandHandler.sendTranslatedMessage(sender, "commands.talent.out_of_range");
+                    return;
+                }
+                // This is small so array is not needed imo
+                skillId = skillDepot.getSkills().get(0);
+                setTalentLevel(sender, avatar, skillId, newLevel);
+                skillId = skillDepot.getSkills().get(1);
+                setTalentLevel(sender, avatar, skillId, newLevel);
+                skillId = skillDepot.getEnergySkill();
+                setTalentLevel(sender, avatar, skillId, newLevel);
```
Comment (Birdulon) [inline src/main/java/emu/grasscutter/command/commands/TalentCommand.java]: general advice:
```suggestion
                setTalentLevel(sender, avatar, skillDepot.getSkills().get(0), newLevel);
                setTalentLevel(sender, avatar, skillDepot.getSkills().get(1), newLevel);
                setTalentLevel(sender, avatar, skillDepot.getEnergySkill(), newLevel);
```

but specifically for this:
```suggestion
                skillDepot.getSkillsAndEnergySkill().forEach(id -> setTalentLevel(sender, avatar, id, newLevel));
```

Comment (Birdulon) [reply src/main/java/emu/grasscutter/command/commands/TalentCommand.java]: oh whoops, `getSkillsAndEnergySkill` also hits alternate sprints

Comment (natsurepo) [reply src/main/java/emu/grasscutter/command/commands/TalentCommand.java]: thanks for the suggestion!

**Review by Birdulon — APPROVED** [review]

**Pull request merged**